README.md

```diff
@@ -0,0 +1,33 @@
+# Firefox Accounts Basket Proxy
+
+[![Build Status: Travis](https://travis-ci.org/mozilla/fxa-basket-proxy.svg?branch=master)](https://travis-ci.org/mozilla/fxa-basket-proxy)
```

Comment:
I _think_ I enabled Travis-CI for this repo, but not sure how to set up coveralls below.